stackit/internal/services/loadbalancer/loadbalancer/resource.go

```diff
@@ -0,0 +1,551 @@
+		"unhealthy_threshold":  "Unhealthy threshold of the health checking.",
+		"target_pools.name":    "Target pool name.",
+		"target_port":          "Identical port number where each target listens for traffic.",
+		"targets":              "List of all targets which will be used in the pool. Limited to 250.",
+		"targets.display_name": "Target display name",
+		"ip":                   "Target IP",
+	}
+
+	resp.Schema = schema.Schema{
+		Description: descriptions["main"],
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Description: descriptions["id"],
+				Computed:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"project_id": schema.StringAttribute{
+				Description: "STACKIT project ID to which the dns record set is associated.",
+				Required:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+				Validators: []validator.String{
+					validate.UUID(),
+					validate.NoSeparator(),
+				},
+			},
+			"external_address": schema.StringAttribute{
+				Description: descriptions["external_address"],
+				Optional:    true,
+				Computed:    true,
+			},
+			"listeners": schema.ListNestedAttribute{
+				Description: descriptions["listeners"],
+				Optional:    true,
+				Computed:    true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"display_name": schema.StringAttribute{
+							Description: descriptions["listeners.display_name"],
+							Optional:    true,
+							Computed:    true,
+						},
+						"name": schema.StringAttribute{
+							Description: descriptions["listeners.display_name"],
+							Computed:    true,
+						},
+						"port": schema.Int64Attribute{
+							Description: descriptions["port"],
+							Optional:    true,
+							Computed:    true,
+						},
+						"protocol": schema.StringAttribute{
+							Description: descriptions["protocol"],
+							Optional:    true,
+							Computed:    true,
+						},
+						"target_pool": schema.StringAttribute{
+							Description: descriptions["target_pool"],
+							Optional:    true,
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"name": schema.StringAttribute{
+				Description: descriptions["name"],
+				Optional:    true,
+				Computed:    true,
+				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(1),
+					stringvalidator.LengthAtMost(63),
+				},
+			},
+			"networks": schema.ListNestedAttribute{
+				Description: descriptions["networks"],
+				Optional:    true,
+				Computed:    true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"network_id": schema.StringAttribute{
+							Description: descriptions["network_id"],
+							Optional:    true,
+							Computed:    true,
+							Validators: []validator.String{
+								validate.UUID(),
+								validate.NoSeparator(),
```

Comment:
You don't need the `NoSeparator`. That's only used for fields that are used to build the internal ID (in this case, `project_id` and `name`). It's so it's guaranteed that, when we split `id` by the separator during the `ImportState` routine, we aren't doing anything we shouldn't